apps/molgenis-components/src/queryEmx2/queryEMX2.test.ts

```diff
@@ -112,7 +112,7 @@ Biobanks(filter: { _or: [ { country: { name: { like: "DE" } } }, { collections:
       .getQuery();
 
     expect(query).toStrictEqual(`{
-Biobanks(filter: { _and: [ { name: { like: "Dresden" } }, { country: { name: { equals: "DE" } } } ] }) {
+Biobanks(filter: { _and: [ { _and: [ { name: { like: "Dresden" } } ] }, { _and: [ { country: { name: { equals: "DE" } } } ] } ] }) {
```

Comment:
why these nested _and?
That has no meaning if the list is 1 item long.

Reply:
or again, probably just an example, then this can be resolved.

Reply:
offtopic, we should remove this query library and write the queries directly OR make sure that library reflects the syntax of graphql.

Reply:
that is why I said offtopic. Maybe I should use another word, I meant to say: not in this PR but just to check our shared vision on this.